src/app/clusters/diagnostic-logs-server/BDXDiagnosticLogsProvider.cpp

```diff
@@ -279,7 +283,11 @@ void BDXDiagnosticLogsProvider::Reset()
 
     if (mDelegate != nullptr)
     {
-        mDelegate->EndLogCollection(mLogSessionHandle);
+        CHIP_ERROR err = mDelegate->EndLogCollection(mLogSessionHandle, error);
+        if (err == CHIP_ERROR_NOT_IMPLEMENTED)
+        {
+            ChipLogError(DeviceLayer, "EndLogCollection not implemented in the delegate. Falling back to default behavior.");
```

Comment:
Again, this does not seem to describe what we really do, which is ignore the return value entirely.

Reply:
@bzbarsky-apple I have removed these unnecessary lines. Please review.